content/200-concepts/200-database-connectors/08-cockroachdb.mdx

```diff
@@ -0,0 +1,71 @@
+
+```prisma file=schema.prisma
+generator client {
+  provider        = "prisma-client-js"
+  previewFeatures = ["cockroachdb"]
+}
+```
+
+## Connection details
+
+CockroachDB uses the PostgreSQL format for its connection URL. See the [PostgreSQL connector documentation](/concepts/database-connectors/postgresql#connection-details) for details of this format, and the optional arguments it takes.
+
+## Type mapping between CockroachDB and the Prisma schema
+
+The CockroachDB connector maps the [scalar types](/concepts/components/prisma-schema/data-model#scalar-fields) from the Prisma [data model](/concepts/components/prisma-schema/data-model) to native column types. These native types are mostly the same as for PostgreSQL — see the [PostgreSQL connector documentation](/concepts/database-connectors/postgresql#type-mapping-between-postgresql-to-prisma-schema) for details. However, there are some limitations, listed below.
+
+> Alternatively, see the [Prisma schema
+> reference](/reference/api-reference/prisma-schema-reference#model-field-scalar-types)
+> for type mappings organized by Prisma type.
+
+### Native type mapping limitations
+
+When introspecting a CockroachDB database, the database types are mapped to Prisma in the same way as for PostgreSQL, with the following limitations:
+
+| CockroachDB (Type \| Aliases) | Prisma    | Supported | Native database type attribute | Notes                                                    |
+| ----------------------------- | --------- | :-------: | :----------------------------- | :------------------------------------------------------- |
+| `money`                       | `Decimal` |  Not yet  | `@db.Money`                    | Supported in PostgreSQL but not currently in CockroachDB |
+| `xml`                         | `String`  |  Not yet  | `@db.Xml`                      | Supported in PostgreSQL but not currently in CockroachDB |
+
+## Other limitations
+
+The following table lists any other limitations of CockroachDB compared to PostgreSQL:
+
+| Issue                                                        | Area   | Notes                                                                                                                                                                                                                                                  |
+| ------------------------------------------------------------ | ------ | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| `Json[]` type not supported                                  | Schema | `Json` arrays are currently not supported in CockroachDB                                                                                                                                                                                               |
```

Comment:
From a grouping perspective it might make sense to put that next to the native types, and just call it "Type mapping limitation" or something like that.

(Could also take GH link for context)

Reply:
Done - let me know if I should use a different phrase to `jsonb arrays` in the first cell